src/codesign-validator.ts

```diff
@@ -46,19 +46,8 @@ export default class CodeSignValidator {
 
     validateMacSignature(fileExtension: string) {
         let promise = utility.defer();
-        let command = `spctl --assess -vv ${this.filePath} --type `;
+        let command = `codesign -dv --verbose=4 ${this.filePath}`;
```

Comment:
does this cmd line tool exist out of the box?

Reply:
the codesign tool is available out of the box